src/app.js

```diff
@@ -55,6 +57,20 @@ function createWindow() {
   WindowState.use('main', mainWindow)
 
   mainWindow.loadURL('https://mail.google.com')
+
+  mainWindow.webContents.on('dom-ready', () => {
+    fs.readFile(
+      path.join(__dirname, '/../static/style.css'),
+      'utf-8',
+      (error, data) => {
+        if (!error) {
+          let formatedData = data.replace(/\s{2,10}/g, ' ').trim()
```

Comment:
@timche In #40 I updated to using Gulp.  I would like to see this whitespace replacement be part of that process using gulp-cssmin.